patches/server/0095-Fix-MC-266334.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: PadowYT2 <me@padow.ru>
+Date: Sun, 21 Jul 2024 15:16:54 +0300
+Subject: [PATCH] Fix MC-266334
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
```

Comment:
Not sure if this should be added, but this was done by git

Reply:
Yeah, this is auto generated by git for compatibility